presto-kafka/src/main/java/com/facebook/presto/kafka/KafkaSplitManager.java

```diff
@@ -93,44 +92,35 @@ public ConnectorSplitSource getSplits(
     {
         KafkaTableHandle kafkaTableHandle = convertLayout(layout).getTable();
         try {
-            SimpleConsumer simpleConsumer = consumerManager.getConsumer(selectRandom(nodes));
-
-            TopicMetadataRequest topicMetadataRequest = new TopicMetadataRequest(ImmutableList.of(kafkaTableHandle.getTopicName()));
-            TopicMetadataResponse topicMetadataResponse = simpleConsumer.send(topicMetadataRequest);
+            HostAddress node = servers.selectRandomServer();
+            String topic = kafkaTableHandle.getTopicName();
 
+            KafkaConsumer<ByteBuffer, ByteBuffer> consumer = consumerManager.createConsumer(Thread.currentThread().getId(), node);
```

Comment:
use thread getName(), instead of getId()

Reply:
Fixed.